app/models/calculator.rb

```diff
@@ -0,0 +1,12 @@
+class Calculator < ApplicationRecord
+  has_many :fields
+  before_validation :set_uuid
+  validates :name, :presence => true
+  validates :name, :format => { :with => /\A[a-zA-Z]+\z/,
+  :message => "Only letters allowed" }
+  validates :name, :length => { :minimum => 2 }
```

Comment:
Format your hashes per https://github.com/rubocop/ruby-style-guide#hash-literals

Reply:
Thanks. I've changed it in my last commit.